index.js

```diff
@@ -85,6 +84,10 @@ app.use(
   }),
 )
 
+function convertToSeconds(microseconds) {
```

Comment:
Do you mean `milliseconds` instead of `microseconds`?

Reply:
I do whoops